data/translations/ja.ts

```diff
@@ -5,14 +5,14 @@
     <name>PictureBox</name>
     <message>
         <source>Press to login</source>
-        <translation type="unfinished"></translation>
+        <translation>押してログイン</translation>
```

Comment:
@kisaragi-hiu
It sounds a bit unnatural, but I'm not sure how to make it natural because I don't know the context...
I found it is used [here](https://github.com/sddm/sddm/blob/develop/components/2.0/PictureBox.qml#L125), but not sure how it is displayed. (Screenshot is much appreciated if you have)

Reply:
It's used in the default Maui theme, shown for users that do not have passwords.

![螢幕截圖_20250126_010258](https://github.com/user-attachments/assets/7b849247-6086-4b6a-b468-6b209d95b4c1)

Reply:
Thank you.
I could not find the best translation, but I suggest adding ここを.
The new Japanese translation literally means "Press **here** to login."

```suggestion
        <translation>ここを押してログイン</translation>
```

Reply:
Applied in [`2aaa80d` (#2036)](https://github.com/sddm/sddm/pull/2036/commits/2aaa80da6995ce3f57d286d4fde92851264779f6)